lib/ui/widgets/custom_list_tile.dart

```diff
@@ -0,0 +1,16 @@
+import 'package:flutter/material.dart';
+
+class CustomListTile extends StatelessWidget {
+  final IconData iconData;
+  final String text;
+
+  CustomListTile(this.iconData,this.text);
```

Comment:
Why these properties are not optional .. it’s safe to make them optional .. give attention in call ... when I tell people to make constructor using {}